packages/card/stories/card.stories.ts

```diff
@@ -229,31 +234,31 @@ export const smallQuiet = (): TemplateResult => {
     `;
 };
 
-export const SlottedTitle = (): TemplateResult => {
+export const SlottedHeading = (): TemplateResult => {
```

Comment:
This changes the name of the visual regression target in `/test/visual/stories.js`. Looks like it's the last missing thing.

Reply:
this is where magic re-naming fails. I can't search the codebase for `SlottedTitle` to find where this is used :(

Reply:
(I mean... not that I did 🙈 )

Reply:
Yes. 💯  Thinking about how to make that not so, but _priorities_... thanks!